openproblems/tasks/denoising/methods/magic.py

```diff
@@ -14,18 +14,35 @@
     code_url="https://github.com/KrishnaswamyLab/MAGIC",
     image="openproblems-python-extras",
 )
+_knn_naive_method = functools.partial(
+    method,
+    paper_name="NA",
```

Comment:
Please fill paper_name and paper_url. For an example of how to fill decorators for baseline methods, see https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/label_projection/methods/baseline.py